Agents/MetOfficeAgent/Dockerfile

```diff
@@ -17,18 +17,30 @@ ENV PYTHONUNBUFFERED=1
 # Install Java
 RUN apt update && apt install -y openjdk-11-jdk-headless git
 
+# Copy resources required to update py4jps later
+COPY ./tmp_base ./tmp_base
+COPY ./tmp_stack ./tmp_stack
+
 # Set the default working directory & copy files to install Python
 WORKDIR /app
 COPY ./README.md .
 COPY ./setup.py .
-COPY ./resources/metoffice.properties ./resources/metoffice.properties
+COPY ./resources/ontop.obda ./resources/ontop.obda
+COPY ./resources/postgresql-42.5.0.jar ./resources/postgresql-42.5.0.jar
```

Comment:
another way is to download this into your image

Reply:
I thought about it, but did not know how to exactly do it. Any guidance?

Reply:
the url is "https://jdbc.postgresql.org/download/postgresql-42.5.0.jar", you can either download it with your code or probably add it in the Dockerfile

Reply:
Now included in commit `fb9e54139d97165fc9e9f033fda4f77af2d4375b`